packages/opc-header/README.md

```diff
@@ -0,0 +1,180 @@
+# opc-header component 👋
+
+![Version](https://img.shields.io/badge/version-0.0.1-blue.svg?cacheSeconds=2592000)
+![Build Status](https://travis-ci.org/dwyl/esta.svg?branch=master)
+[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/1-Platform/op-components/graphs/commit-activity)
+
+A standardized web component based on Lit Element for Red Hat One Platform Header that uses Patternfly Breadcrumb and Links as button HTML component
+
+
+## Usage
+Plain Header
+
+```html
+ <opc-header title="OPC Header"></opc-header>
+```
+
+Header with Breadcrumbs and Links
+```html
+ <opc-header title="OPC Header">
+  <opc-header-breadcrumb slot="breadcrumb"></opc-header-breadcrumb>
+  <opc-header-links slot="links"></opc-header-links>
+ </opc-header>
+```
+For custom options, 
+- use the `opcHeaderBreadcrumb` setter function to set the header breadcrumb
+- use the `opcHeaderLinks` setter function to set the header links
+
+```js
+  const breadcrumbs = [
+    {
+      "name":"Section Home",
+      "href":"#"
+    },
+    {
+      "name":"Section Landing",
+      "href":"#"
+    }];
+
+  const links = [
+    {
+      "name":"Link1",
+      "href":"#",
+      "icon":"fa-play-circle"
+    },
+    {
+      "name":"Link2",
+      "href":"#",
+      "icon":"fa-question-circle"
+    },
+    {
+      "name":"Link3",
+      "href":"#",
+      "icon":"fa-file"
+    }];
+
+  document.querySelector("opc-header-breadcrumb").opcHeaderBreadcrumb = breadcrumbs;
+  document.querySelector("opc-header-links").opcHeaderLinks = links;
+```
+
+Header with color themes
+```html
+ <opc-header title="OPC Header" theme="dark">
+  <opc-header-breadcrumb slot="breadcrumb"></opc-header-breadcrumb>
+  <opc-header-links slot="links"></opc-header-links>
+ </opc-header>
+```
+
+Header with lightDOM
+```html
+ <opc-header title="OPC Header" theme="red">
```

Comment:
the **title** is a native global attribute for an element... can we change it another name

Reply:
@gisumit yes makes sense